docs/sections/contributor_guide/developer_setup.rst

```diff
@@ -1,5 +1,183 @@
+| ``make test``       | Equivalent to                                              |
+|                     | ``make lint && make typecheck && make unittest``           |
+|                     |                                                            |
+|                     | Checks defined CLI scripts                                 |
++---------------------+------------------------------------------------------------+
+
+
+Note that ``make format`` is never run automatically, to avoid
+reformatting under-development code in a way that might surprise the
+developer. A useful development idiom is to periodically run
+``make format && make test`` to perform a full code-quality sweep
+through the code.
+
+The ``make test`` command is also automatically executed when ``conda``
+builds a ``uwtools`` package, so it is important to periodically run
+these tests during development and, crucially, before merging changes,
+to ensure that the tests will pass when CI builds the ``workflow-tools``
+code.
+
+
+The order of the targets above is intentional, and possibly useful:
+
+   * ``make format`` will complain about certain kinds of syntax errors that
+     would cause all the remaining code-quality tools to fail (and may change
+     line numbers reported by other tools, if it ran after them).
+   * ``make lint`` provides a good first check for obvious errors and
+     anti-patterns in the code.
+   * ``make typecheck`` offers a more nuanced look at interfaces between 
+     functions, methods, etc. and may spot issues that would cause 
+     ``make unittest`` to fail.
+
+
+In addition to the ``make devshell`` command, other ``make`` targets
+are available for use *outside* a development shell, i.e. from the base
+conda environment (requires presence of the ``condev`` package):
+
+
++------------------+-------------------------------------------------------+
+| Command          | Description                                           |
++==================+=======================================================+
+| ``make env``     | Creates a conda environment based on the ``uwtools``  |
+|                  | code                                                  |
++------------------+-------------------------------------------------------+
+| ``make meta``    | Update ``recipe/meta.json`` from ``recipe/meta.yaml`` |
++------------------+-------------------------------------------------------+
+| ``make package`` | Builds a ``uwtools`` conda package                    |
++------------------+-------------------------------------------------------+
+
+
+These targets work from the code in its current state in the clone.
+``make env`` calls ``make package`` automatically to create a local
+package, then builds an environment based on the package.
+
+Building condev locally
+-----------------------
+
+As an alternative to installing a prebuilt package from
+`anaconda`_, the ``condev`` package can be
```

Comment:
This is the line where "anaconda" appears. These texts need to be identical or the hyperlink will not be found.

`.. _anaconda: https://anaconda.org/maddenp/condev`

Did you want me to update this documentation to something else 
i.e. " `anaconda`_, the ``condev`` package " currently appears. I can make these words hyperlink together if it more accurately states the link. 

" `ananconda, the condev package`_ "

Reply:
Maybe I covered this in my previous comment, but I'm thinking that a reader might think "What on earth is 'a prebuilt package from anaconda'?" but if that text was hyperlinked to https://anaconda.org/maddenp/condev, they could visit that page and get an idea of what we're talking about. So, I think you and I are talking about the same solution, but I think the words to hyperlink together are "a prebuilt package from anaconda".